modules/hcp-support-matrix.adoc

```diff
@@ -84,7 +84,14 @@ The following table maps {mce-short} versions to the hosted cluster versions tha
 [id="hcp-matrix-platform_{context}"]
 == Hosted cluster platform support
 
-The following table indicates which {product-title} versions are supported for each platform of {hcp}. In the table, _Management cluster version_ refers to the {product-title} version where the {mce-short} is enabled:
+The following table indicates which {product-title} versions are supported for each platform of {hcp}.
+
+[IMPORTANT]
+====
+For {ibm-power-title} and {ibm-z-title}, you must run the control plane on machine types based on 64-bit x86 architecture and node pools on {ibm-power-title} or {ibm-z-title}.
+====
+
+In the table, _Management cluster version_ refers to the {product-title} version where the {mce-short} is enabled:
```

Comment:
Per ISG, column names in a text do not need highlighting. 

> If the name is difficult to distinguish from the
surrounding text, enclose the name in double quotation
marks.

I think there we wouldn't need `"` either. But up to you.

```suggestion
In the following table, Management cluster version refers to the {product-title} version where the {mce-short} is enabled:
```

I understand this is an existing content that was moved. But just to ensure that this adheres guidelines, you could consider this comment.